### ISSUE TITLE
add an "open all" to notification email

### ISSUE THREAD

**Issue by lsabor** [Metaculus/metaculus#1688]
[User exmateriae suggests:](https://www.metaculus.com/notebooks/935/platform-feature-suggestions/#comment-239120)

>Something I would like (not sure how doable it is) would be a "open all" link in the notification mails. When I get a mail for comments or significant changes with 13 different questions, I would rather open the m all and check directly on the website.

**Comment by lsabor** [comment]
I don't think we have an official "this is to be decided yes or no" label, so I put Do not merge yet in place of that.

**Comment by aseckin** [comment]
Not sure if I understand how this would work - would a single button open 13 tabs at once? Does not sound feasible or necessary
